scripts/makeNetwork.py

```diff
@@ -138,7 +138,8 @@ def qemuArchNetworkConfig(i, arch, n):
         if arch == "arm":
             return "-device virtio-net-device,netdev=net%(I)i -netdev socket,id=net%(I)i,listen=:200%(I)i" % {'I': i}
         else:
-            return "-net nic,vlan=%(VLAN)i -net socket,vlan=%(VLAN)i,listen=:200%(I)i" % {'I': i, 'VLAN' : i}
+            #workaround for qemu >= 2.1.1 tested on Ubuntu 19 with qemu 3.1
+            return "-netdev user,id=n%(VLAN)i -device e1000,netdev=n%(VLAN)i -nic socket,listen=:200%(I)i" % {'I': i, 'VLAN' : i}
```

Comment:
`%(VLAN)i` should be changed to `%(I)i`, and you can drop the `VLAN` field from the dictionary.

Reply:
Fixed  in the commit `bf57905`.